scriptmodules/libretrocores/lr-flycast.sh

```diff
@@ -82,7 +72,6 @@ function configure_lr-flycast() {
 
     local def=0
     isPlatform "kms" && def=1
-    # segfaults on the rpi without redirecting stdin from </dev/null
```

Comment:
Can we keep a comment about the redirect - as it's not obvious from the code why it's needed.

Reply:
Yes, initially I removed the redirection (there was a commit about input being done on a separate thread), but it looks like it's still needed.